### ISSUE TITLE
Console

### ISSUE THREAD

**Issue by wnederhof** [philc/vimium#2238]
First of all, thank you for Vimium, I use it on a daily basis, and I love it!

However, it would be nice if all .js/.css-files would be packed into one or two files. As I am using Google Chrome's console a lot, the Network section shows many .js and .css files from Vimium, thereby making debugging other projects more difficult while using Vimium.

(I have seen the issue #1838 which is closed. However, it is not fixed on my browser, having downloaded the plugin from Google Chrome's store just recently.)

**Comment by smblott-github** [comment]
If you disable Vimium on the site you're working on, then you shouldn't see any Vimium network requests on the console. (It works for me.)

Disable Vimium via the page popup.  See [here](https://github.com/philc/vimium/wiki/Disabling-Vimium).

**Comment by wnederhof** [comment]
Yes, of course, but then I cannot use Vimium during development. :)

**Comment by slavaGanzin** [comment]
@wnederhof 
Was curious to find solution. Looks like this can't be set to default but it works:
Set Filter fileld (on Network tab) to:
```-scheme:chrome-extension```

I think this should be documented in vimium. Even if this is a chrome problem(which they don't want to fix)

**Comment by philc** [comment]
Thankfully, this is now a built-in setting in Chrome's console:

<img width="700" height="253" alt="Image" src="https://github.com/user-attachments/assets/0dad5f4c-c77d-4af9-8bdf-9e3fac46c366" />
